packages/aws-rfdk/lib/deadline/lib/render-queue.ts

```diff
@@ -519,6 +524,15 @@ export class RenderQueue extends RenderQueueBase implements IGrantable {
     child.node.addDependency(this.taskDefinition);
   }
 
+  public addSecurityGroups(securityGroups: RenderQueueSecurityGroupsOptions) {
```

Comment:
I suggest splitting this in two, to make the interface simpler in Python:
1. `addFrontendSecurityGroups( array of SGs )` -> Add the security groups to the load balancer.
2. `addBackendSecurityGroups( array of SGs )` -> Add the security groups to the backend components (currently just the RCS's ASG).